rudder-webapp/SOURCES/Makefile

```diff
@@ -258,12 +259,13 @@ install: build
 	cp -a rudder-jetty-base $(DESTDIR)/opt/rudder/etc
 	
 	# Init script
-	install -m 755 jetty/bin/jetty.sh $(DESTDIR)/opt/rudder/bin/
+	install -m 755 jetty/bin/$(JETTY_SCRIPT) $(DESTDIR)/opt/rudder/bin/rudder-jetty.sh
 	install -m 644 rudder-jetty.default $(DESTDIR)/etc/default/rudder-jetty
 	install -m 644 rudder-jetty.conf $(DESTDIR)/opt/rudder/etc/
 	install -m 644 rudder-jetty $(DESTDIR)/opt/rudder/etc/server-roles.d/
 
 	install -m 755 rudder-webapp-postinst $(DESTDIR)/opt/rudder/share/package-scripts/
+	install -m 644 rudder-jetty.service $(DESTDIR)/lib/systemd/system/
```

Comment:
/usr/lib, I'll make a PR for agent in 4.3

Reply:
how come that it works for the agent ?